package.json

```diff
@@ -1,31 +1,30 @@
 {
   "name": "@liftedinit/many-js",
-  "version": "0.1.0-alpha.7",
+  "version": "0.1.0-alpha.8",
   "homepage": "https://github.com/liftedinit/many-js",
   "main": "dist/index.js",
   "files": [
     "dist"
   ],
   "dependencies": {
-    "@types/crc": "^3.4.0",
-    "@types/jest": "^27.0.3",
-    "@types/node": "^12.20.37",
-    "@types/node-forge": "^0.10.10",
-    "@types/uuid": "^8.3.3",
-    "@typescript-eslint/parser": "^5.7.0",
+    "@types/crc": "^3.8.0",
+    "@types/jest": "^29.2.6",
+    "@types/node": "^18.11.18",
+    "@types/node-forge": "^1.3.1",
+    "@types/uuid": "^9.0.0",
+    "@typescript-eslint/parser": "^5.48.0",
```

Comment:
Let's take this opportunity to move all the `@types` stuff into `devDependencies`.